src/main/java/com/health/controller/HomeController.java

```diff
@@ -4543,17 +4570,121 @@ public String createPackagePost(HttpServletRequest req, Model model, Principal p
     }
 
     /******************************************
-     * AssTutorial on Week And Package End
+     * Assign Tutorial on Week And Package End
      ********************/
 
+    private int extractInteger(String str) {
+        // Use regular expression to find all digits in the string
+        String numberStr = str.replaceAll("\\D+", ""); // \\D+ matches all non-digit characters
+        if (!numberStr.isEmpty()) {
+            return Integer.parseInt(numberStr);
+        } else {
+            return 0;
+        }
+    }
+

```

Comment:
don't define the same method again. use the previous method instead. make that method public static.